proofreader-api-playground/script.js

```diff
@@ -68,12 +68,14 @@ const button = popover.querySelector('button');
     e.preventDefault();
 
     // Use existing proofreader instance or create new instance.
-    proofreader = proofreader || await self.Proofreader.create({
-      includeCorrectionTypes: true,
-      includeCorrectionExplanations: true,
-      expectedInputLanguagues: ['en'],
-      correctionExplanationLanguage: 'en',
-    });
+    if (proofreaderAPISupported) {
+      proofreader = proofreader || await self.Proofreader.create({
+        includeCorrectionTypes: true,
+        includeCorrectionExplanations: true,
+        expectedInputLanguagues: ['en'],
+        correctionExplanationLanguage: 'en',
+      });
+    }
```

Comment:
I wonder if the below would be simpler to read:

```suggestion
    if (!proofreader && proofreaderAPISupported) {
      proofreader = await self.Proofreader.create({
        includeCorrectionTypes: true,
        includeCorrectionExplanations: true,
        expectedInputLanguagues: ['en'],
        correctionExplanationLanguage: 'en',
      });
    }
```